src/mofaflex/_core/mofaflex.py

```diff
@@ -133,6 +136,12 @@ class ModelOptions(_Options):
     nonnegative_weights: dict[str, bool] | bool = False
     """Non-negativity constraints for weights for each view (if dict) or for all views (if bool)."""
 
+    guiding_vars_weight_priors: dict[str, str] | Literal["Normal"] | None = "Normal"
+    """Weight prior for each guiding variable (if dict) or for all guiding variables (if str)."""
```

Comment:
Since only the Normal prior is implemented currently, does it make sense to have this parameter?